src/components/WidgetHugger.vue

```diff
@@ -87,7 +92,7 @@ const hoveringWidgetOrOverlay = computed(() => hoveringOverlay.value || hovering
 
 // Put the widget into highlighted state when in edit-mode and hovering over it
 watch([hoveringWidgetOrOverlay, allowMoving], () => {
-  widgetStore.widgetManagerVars(widget.value.hash).highlighted = hoveringWidgetOrOverlay.value && allowMoving.value
+  widgetStore.widgetManagerVars(widget.value.hash).highlighted = hoveringWidgetOrOverlay.value
```

Comment:
Any reason to change this line?

Reply:
Since the Custom Widget Base can only be dragged around by the icon on its upper-left corner (otherwise it wouldn't be possible to drag the components inside it), the highlight feature on those widgets would only be shown while dragging.

In my tests, I couldn't see any collateral effect of removing the 'allowMoving.value' part. Is it affecting the other widgets behavior?